repos/fdbt-site/src/pages/api/selectSalesOfferPackage.ts

```diff
@@ -71,9 +79,17 @@ export const sanitiseReqBody = (
 export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
     try {
         const multipleProductAttribute = getSessionAttribute(req, MULTIPLE_PRODUCT_ATTRIBUTE);
-        const products = multipleProductAttribute
-            ? multipleProductAttribute.products
-            : [{ productName: 'product', productPrice: '' }];
+        const ticket = getSessionAttribute(req, MATCHING_JSON_ATTRIBUTE);
+        const matchingJsonMetaData = getSessionAttribute(req, MATCHING_JSON_META_DATA_ATTRIBUTE);
+
+        let prod = [];
```

Comment:
Could we change this to be `const products = []` and inside the if block, instead of doing` prod = <something>`, we can do `products.push({ productName: ticket.products[0].productName, productPrice: ticket.products[0].productPrice })` etc.